============================= test session starts ==============================
platform darwin -- Python 3.8.13, pytest-7.1.2, pluggy-1.0.0 -- /opt/anaconda3/envs/py38/bin/python
cachedir: .pytest_cache
rootdir: /Users/wladimir/Dropbox/Yachay_Tech/Semestre2_2022/Docencia/Fisica_Computacional_2/Lectures/computational-physics-2/unit-5
plugins: anyio-3.5.0
collecting ... collected 2 items

test_simple.py::test_multiply PASSED                                     [ 50%]
test_simple.py::test_multiply2 FAILED                                    [100%]

=================================== FAILURES ===================================
________________________________ test_multiply2 ________________________________

    def test_multiply2():
>       assert multiply(5, 6) == 2
E       assert 30 == 2
E        +  where 30 = multiply(5, 6)

test_simple.py:8: AssertionError
=========================== short test summary info ============================
FAILED test_simple.py::test_multiply2 - assert 30 == 2
========================= 1 failed, 1 passed in 0.08s ==========================
